Reply/app/src/main/res/drawable/ic_toggle_star.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+  Copyright (c) 2021 Google Inc.
+
+  Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+  in compliance with the License. You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software distributed under the License
+  is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+  or implied. See the License for the specific language governing permissions and limitations under
+  the License.
+  -->
+<selector xmlns:android="http://schemas.android.com/apk/res/android">
+
+  <item
```

Comment:
For this ColorStateList, you only need to specify the `state` for the `true` case. When unchecked, you can leave off the state_checked attribute. An item with no state will be used as the "default" case. So this can look like:

```
<selector...>
    <item android:drawable="@drawable/ic_star_filled" android:state_checked="true" />
    <item android:drawable="@drawable/ic_star_border" />
</selector>
```

Reply:
Also, same comment as above with formatting. Can you follow this example? https://github.com/material-components/material-components-android-examples/blob/develop/Reply/app/src/main/res/color/color_navigation_drawer_menu_item.xml